portfolio/src/main/java/com/google/sps/servlets/QuoteGenerator.java

```diff
@@ -9,10 +13,48 @@
 /** Handles requests sent to the /quote URL endpoint */
 @WebServlet("/quote")
 public class QuoteGenerator extends HttpServlet {
-
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>\"Life changes fast. Life changes in the instant.\" - Joan Didion</h1>");
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response)
+      throws IOException {
+    // Get a random quote
+    Quote randomQuote = this.getRandomQuote();
+
+    // Parse the quote and return the json output
+    final Gson gson = new Gson();
+    final String json = gson.toJson(randomQuote);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Build the quotes using the author name and quote text.
+   *
+   * @param authorName Name of the quote author
+   * @param quoteText  The quote text
+   * @return A Quote object with the author's name and the text
+   */
+  private Quote buildQuote(final String authorName, final String quoteText) {
+    final Quote quoteInfo = new Quote(authorName, quoteText);
+    return quoteInfo;
+  }
+
+  /**
+   * Return one random quote from an ArrayList of quotes
+   *
+   * @return a random quote
+   */
+  private Quote getRandomQuote() {
+    // Add all to the quotes ArrayList
```

Comment:
You can remove this comment. I think it is clear that you are creating a list of quotes